experimental/generation/generator/packages/library/templates/standard/en-us/form.en-us.lg

```diff
@@ -110,4 +110,88 @@ You can also ask for over all help, cancel the whole form or skip a step.```
 
 > Create new required properties where expected properties are moved to the end
 # skipExpectedProperties()
-- ${union(where($requiredProperties, property, !contains($expectedProperties, property)), where($expectedProperties, property, contains($requiredProperties, property)))}
\ No newline at end of file
+- ${union(where($requiredProperties, property, !contains($expectedProperties, property)), where($expectedProperties, property, contains($requiredProperties, property)))}
+
+# number_Value(val)
+ - IF: ${val}
+ - ${val}
+ - ELSE:
+ - no value
+
+# integer_Value(val)
+ - IF: ${val}
+ - ${val}
+ - ELSE:
+ - no value
+
+# percentage_Value(val)
+ - IF: ${val}
+ - ${val}
+ - ELSE:
+ - no value
+
+# string_Value(val)
+ - IF: ${val}
+ - ${val}
+ - ELSE:
+ - no value
+
+# url_Value(val)
+ - IF: ${val}
+ - ${val}
+ - ELSE:
+ - no value
+
+# phonenumber_Value(val)
+ - IF: ${val}
+ - ${val}
+ - ELSE:
+ - no value
+
+# personName_Value(val)
+ - IF: ${val}
+ - ${val}
+ - ELSE:
+ - no value
+
+# keyPhrase_Value(val)
+ - IF: ${val}
+ - ${val}
+ - ELSE:
+ - no value
+
+# email_Value(val)
+ - IF: ${val}
+ - ${val}
+ - ELSE:
+ - no value
+
+# utterance_Value(val)
+ - IF: ${val}
+ - ${val}
+ - ELSE:
+ - no value
+
+ # dimension_Value(val)
+ - IF: ${val}
+ - ${val.number}
+ - ELSE:
+ - no value
+
+# age_Value(val)
+ - IF: ${val}
+ - ${val.number}
+ - ELSE:
+ - no value
+
+ # money_Value(val)
+ - IF: ${val}
+ - ${val.number}
+ - ELSE:
+ - no value
+
+  # temperature_Value(val)
+ - IF: ${val}
+ - ${val.number}
+ - ELSE:
+ - no value
```

Comment:
I think I would prefer that these were in templates specific to each entity type, i.e. dimension would include the .lg file for this.  That keeps things more modular.